src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+
+//    public void test() throws IOException, DukeException {
+//        assertEquals(new duke.DukeException("OOPS!!! The description of a todo cannot be empty."),
+//                new Parser().interpret("todo", new duke.TaskList(), new duke.Storage("data/tasks.txt")));
+//    }
+}
```

Comment:
Why is everything commented out? Perhaps this file should be deleted if it is no longer needed.